packages/components/src/TabBar/TabBar.scss

```diff
@@ -10,13 +10,62 @@
 	background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M8 13.6l-8-8L2.6 3 8 8.4 13.4 3 16 5.6z" fill="#{hex-to-url($color)}"/></svg>');
 }
 
+$underline-height: 2px;
+
 .tc-tab-bar {
 	white-space: nowrap;
 	overflow: hidden;
 
 	& &-item {
 		float: none;
 		display: inline-block;
+
+		&-icon {
+			margin-right: $padding-smaller;
+			width: $svg-sm-size;
+			height: $svg-sm-size;
+			font-size: $svg-sm-size;
+		}
+
+		button {
+			padding: $padding-smaller $padding-small;
+			padding-bottom: calc(#{$padding-smaller} + #{$underline-height});
+			color: $black;
+
+			max-width: 200px;
```

Comment:
no rem?

Reply:
It's not clear enought for me why using rem right over here for width size, such as rem depends on font-size. And in fontify is max-width provided with exact value 200px. In this case if font-size will change, max-width size will also change.

Reply:
+1 with @vbocquet 
Guidelines do not respect UX recommendations for A11Y issues imho.
By default using rem unit will not block you UI when you will zoom in. 
Mixing rem and px is not a good idea at all, 1rem is designed to be an equivalent of 10px so i's easy to not block zoomed viewport.